### ISSUE TITLE
CKEditor toolbar refresh after recreate

### ISSUE THREAD

**Issue by cmak9149** [ckeditor/ckeditor4#4240]
Our page has one language selector and two rich text fields using a sharedspace plugin. When the language change, we will call CKEDITOR.replace() again for the fields, using new language. The ckeditor toolbar will remain in old language, until a rich text fields (associated with the CKEditor) has the focus. 

To overcome this problem, I programmatically has the first ckeditor field has the focus. This is not a good solution because we change the focus from "language selector" to the first rich text field. Worst of all the first rich text field is not the first logic control to have the focus. Force to change focus around can lead to usability issue and bad user experience. 

In short, I need a way to refresh CKEditor after they are recreated, without the rich text field has the focus.

**Comment by Dumluregn** [comment]
Hello, please provide us a reproduction sample of your issue using one of the following links:

* CodePen: https://codepen.io/Comandeer/pen/ExaMgpz?editors=1010
* JSFiddle: https://jsfiddle.net/Comandeer/d6ey8a2w
* JSBin: https://jsbin.com/keqekef/1/edit?html,js,output
* StackBlitz: https://stackblitz.com/edit/js-vcqfw3

Without it it's not clear what exactly is the problem and how the solution should look like.

**Comment by cmak9149** [comment]
Thanks expert. We found a work around by destroy the ckeditor first. That seems to work. Thanks.
